eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -66,6 +66,7 @@ jobs:
       - script: >-
           dotnet test eng/service.proj --filter "(TestCategory!=Manually) & (TestCategory!=Live)" --framework $(TestTargetFramework)
           --logger "trx;LogFileName=$(TestTargetFramework).trx" --logger:"console;verbosity=normal"
+          --blame-crash-dump-type full
```

Comment:
I couldn't get a crash dump to generate successfully on Linux. I'm unsure if it will work on other OSes, but at least this flag generates a sequence file when there is a crash which makes it easier to pinpoint the test that was running at the time of the crash.